rllib/tests/test_gpus.py

```diff
@@ -110,9 +113,119 @@ def test_gpus_in_local_mode(self):
 
         ray.shutdown()
 
+    def test_larger_train_batch_size_multi_gpu_train_one_step(self):
```

Comment:
This test will be super slow because you will need to collect 1e9 samples per iteration before doing grad updates. 

In order to trigger oom, just create a super large model that will oom even with small train batch size (e.g 1024) but is fine with say sgd size of 2. Or alternatively you can create a fake env with huge obs spaces (size of 1e9) you network can be small relatively small (just take the first 128 dims and pas them through an fc layer with input size of 128 to produce the actions). There are other ways to make this run super quick without having to wait for the env.

Reply:
Good one! Thanks

Reply:
As per our discussion offline, we'll just keep the proposed unit test and get rid of the super slow release test.